pkg/bpf/bpffs.go

```diff
@@ -105,6 +106,19 @@ func OpenAfterMount(m *Map) error {
 	return nil
 }
 
+//IsBpffs check if the path is a valid bpf filesystem
+func IsBpffs(path string) bool {
+	// This is the value of the BPF Filesystem. If is into the container the
+	// mountpoint doesn't provide enough information. Defined on uapi/linux/magic.h
+	magic := uint32(0xCAFE4A11)
+	var fsdata syscall.Statfs_t
+	if err := syscall.Statfs(path, &fsdata); err != nil {
```

Comment:
Please use the Statfs function from golang.org/x/sys/unix here instead of the one from syscall. Please see #588 for reference.

Reply:
@tklauser thanks, fixed.